src/ci/bin/test.inc.sh

```diff
@@ -633,6 +637,17 @@ cromwell::private::install_wait_for_it() {
     chmod +x "$CROMWELL_BUILD_WAIT_FOR_IT_SCRIPT"
 }
 
+cromwell::private::install_git_secrets() {
```

Comment:
If we already have this, do we need the `-f` in the minnie_kenny script?

Reply:
Chatted IRL. This section of bash-code only installs the `git-secrets` executable into the `$PATH`. It does not add git-secrets to the git hooks, no use the secrets definitions in minnie-kenny.gitconfig. Those are handled elsewhere by running `minnie-kenny.sh -f`.